test/e2e/specs/templates.test.js

```diff
@@ -60,12 +62,12 @@ describe( 'templates', () => {
 		const STANDARD_FORMAT_VALUE = '0';
 
 		async function setPostFormat( format ) {
+			await switchToAdminUser();
```

Comment:
Switching user will not actually do anything if we're running as the admin user. We'll only switch if we're running as a different user, because the author role cannot manage plugins.

Reply:
That's good to know; could you leave a comment explaining that in the code? 😄